static/js/stream_data.js

```diff
@@ -18,6 +18,12 @@ exports.clear_subscriptions = function () {
 exports.clear_subscriptions();
 
 exports.is_active = function (sub) {
+    if (exports.subscribed_subs().length <= 30) {
+        // If a user doesn't have many streams, then
+        // we consider them all "active", since there's
+        // no point in hiding ones that are dormant.
+        return true;
+    }
```

Comment:
This is probably not the best perf-wise if you have like 1K streams in the organization.  But probably not that big of a deal since it's (total * subscribed).

Reply:
Huh, it is part of a sort, so I think I want to tweak this.